main_test.go

```diff
@@ -16,34 +16,32 @@ func TestFindRoot(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-
-	expect := filepath.Join(wd, "_testdata", "rootfind")
-
-	got1, err := findProjectRoot(expect)
+	want := filepath.Join(wd, "_testdata", "rootfind")
+	got1, err := findProjectRoot(want)
 	if err != nil {
 		t.Errorf("Unexpected error while finding root: %s", err)
-	} else if expect != got1 {
-		t.Errorf("findProjectRoot directly on root dir should have found %s, got %s", expect, got1)
+	} else if want != got1 {
```

Comment:
It's much more common.

- https://github.com/golang/go/search?q=want
- https://github.com/golang/go/search?q=expect